ably-ios/ARTClientOptions.h

```diff
@@ -14,24 +14,32 @@ ART_ASSUME_NONNULL_BEGIN
 
 @interface ARTClientOptions : ARTAuthOptions
 
-@property (readonly, getter=getRestHost) NSString *restHost;
-@property (readonly, getter=getRealtimeHost) NSString *realtimeHost;
+@property (readwrite, strong, nonatomic, getter=getRestHost) NSString *restHost;
```

Comment:
No, scratch that. I'm drunk.

"_5-line code review vs 5000-line code review_" ;)
![she_didn_t_know_what_to_do_](https://cloud.githubusercontent.com/assets/3541185/12941623/360088ea-cfcc-11e5-8669-f1b317d1b373.gif)